docs/building-modules/querier.md

```diff
@@ -15,18 +15,39 @@ A `Querier` designates a function that processes [`queries`](./messages-and-quer
 
 The `querier` type defined in the Cosmos SDK specifies the typical structure of a `querier` function:
```

Comment:
I'd rename this file to `queries.md`, ditto the title

Reply:
This file is more about module query services (and legacy queriers) than queries in themselves, so what about renaming it to `query-services.md` instead?

Reply:
I like it